Atomic/util.py

```diff
@@ -6,10 +6,18 @@
 import os
 import selinux
 from .client import get_docker_client
+from yaml import load as yaml_load
+
 """Atomic Utility Module"""
 
 ReturnTuple = collections.namedtuple('ReturnTuple',
                                      ['return_code', 'stdout', 'stderr'])
+custom_conf = os.environ.get('ATOMIC_CONF')
+
+if custom_conf is None:
+    ATOMIC_CONF = '/etc/atomic.conf'
+else:
+    ATOMIC_CONF = custom_conf
 
```

Comment:
We have decided to make this use /etc/atomic.d  So people can drop multiple config files into this directory.